pages/index.tsx

```diff
@@ -29,7 +29,7 @@ export default function Home() {
 
 	return (
 		<div className='overflow-hidden'>
-			<Navbar ctaLink={chromeExtensionLink} transparent={false} />
+			<Navbar ctaLink={chromeExtensionLink} transparent={true} />
```

Comment:
Why true here? Wouldn't we want to show it on index page as well if extension is not installed?

Reply:
Or this "transparent" is a different setting. It is related to the navbar being transparent on the landing page so that the image behind it is visible. It is a styling thing. 
In this PR, I have just reverted the meaning of true/false in this prop. Earlier, `transparent: true` meant that the Navbar must NOT be transparent. Which was not intuitive.